src/components/Selection.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import Parse from "parse";
```

Comment:
Here and other places where you import Parse:
Consider making a separate .js file with all your parse API functionality together.
could call it api.js :D